### DISCUSSION TITLE
Filtering packages by license

### DISCUSSION THREAD

**Discussion by Pantyhose-X** [topgrade-rs/topgrade#12] [Q&A] [answered]
**Solutions**

Filter arch Linux [packages](https://archlinux.org/packages/) packages by License

Now, all Linux software Repositories had a serious security hazard. Packages manager Do not comply with **Open Source** software,
Arch linux and manjaro official software repositories  have begun to promote closed source software and Dependencies. Such as Opera Browser. This is a very dangerous behaviour that does not open source.

Filter All No Source Code License

**Comment by s34m** [comment, marked as answer]
In this case it's the responsibility of the user to block the use of non open source software on a system level through correctly setting up the system. We will not take any side in the discussion for or against closed source software and will therefor not block any non open source code.

**Reply by Pantyhose-X** [reply]
> In this case it's the responsibility of the user to block the use of non open source software on a system level through correctly setting up the system. We will not take any side in the discussion for or against closed source software and will therefor not block any non open source code.

How do I do it ? I didn't find any tutorials on the internet about it

**Reply by s34m** [reply]
On arch linux use one of the Libre forks instead for example. And for the AUR you have to take care which packages you install. Else it's not possible on arch linux.
